src/views/blocks/rsvp/icon.php

```diff
@@ -18,5 +18,5 @@
 ?>
 <div class="tribe-block__rsvp__icon">
 	<?php $this->template( 'blocks/rsvp/icon-svg' ); ?>
-	<?php esc_html_e( 'RSVP', 'event-tickets' ) ?>
+	<?php echo tribe_get_rsvp_label_singular( basename( __FILE__ ) ) ?>
```

Comment:
:warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')'

<sub>hash: e2082571087427185eadc24bf996b0a5</sub>